js/bybit.js

```diff
@@ -51,8 +51,9 @@ module.exports = class bybit extends Exchange {
                 'fetchTrades': true,
                 'fetchTransactions': undefined,
                 'fetchWithdrawals': true,
-                'setMarginMode': true,
+                'loadTimeDifference': true,
```

Comment:
not sure if this should be in here

Reply:
I think we could unify this in the base class.

Reply:
fetchTime isn't implemented on every exchange

Reply:
To clarify: `loadTimeDifference` should not be in the `has` property of bybit and we should unify the loadTimeDifference implementation in the base class (probably in a separate PR).